docs/developer-docs/latest/setup-deployment-guides/deployment.md

```diff
@@ -1,3 +1,8 @@
+---
+title: Deployment - Strapi Developer Documentation
+description: The following documentation covers how to develop locally with Strapi and deploy Strapi with various hosting options.
```

Comment:
I would suggest against using `the following documentation` and instead swap to something like `Learn how to develop locally with Strapi and deploy Strapi with various hosting options.`